packages/utils/navigation.ts

```diff
@@ -43,6 +44,7 @@ export type RootStackParamList = {
   LaunchpadERC20: undefined;
   LaunchpadERC20Tokens?: { network?: string };
   LaunchpadERC20CreateToken: { step?: number };
+  LaunchpadERC20ManageToken: { network?: string; token: Token };
```

Comment:
```suggestion
  LaunchpadERC20ManageToken: { token: string };
```
we shouldn't use a networkId and a token object as screen params but instead a network-prefixed id that identifies this token (see `getNetworkObjectId`/`parseNetworkObjectId`) so we have a proper permalink